Documentation/Testing.md

```diff
@@ -0,0 +1,19 @@
+# Getting started with testing
+
+Most tests can be run without any special commands or processes. Running the following command from the project's root directory will restore packages, build the solution, and run the full CoreWCF test suite:
+
+    dotnet test
```

Comment:
I think you need to specify the path to the .sln file as we have 2 solution files and they are in the src folder.

Reply:
Added the solution path to the command